[ONNX] Update pytorch/onnx docs for new export API args (#39802)
pytorch/pytorch: docs/source/onnx.rst

            return torch.ops.custom_ops.foo_forward(input1, input2, self.attr1, self.attr2)

    model = FooModel(attr1, attr2)

>     torch.onnx.export(model, (dummy_input1, dummy_input2), 'model.onnx', custom_opsets={"custom_ops": 2})

Reviewer: ```suggestion
    torch.onnx.export(model, (dummy_input1, dummy_input2), 'model.onnx', custom_opsets={"custom_domain": 2})
```
Since the keys are domain names

    assert [torch.allclose(o, torch.tensor(o_ort)) for o, o_ort in zip(out, out_ort)]

Use external data format
------------------------
``use_external_data_format`` argument in export API enables export of models in ONNX external
data format. With this option enabled, the exporter stores some model parameters in external
binary files, rather than the ONNX file itself. These external binary files are stored in the

> same location as the ONNX file. A string specifying where to store the files should

Reviewer: ```suggestion
same location as the ONNX file. Argument 'f' must be a string specifying the location of the model. ```
